### PR TITLE
Okta IDX | Add documentation on how to set up Hoppscotch for the Okta IDX API

### PR DESCRIPTION
**Pull request by coldlink**
## What does this change?

I've created a [Hoppscotch](https://hoppscotch.com/) collection and environment which makes it easier for other developers to start interfacing with the Okta IDX API directly.

This PR adds docs on how to set this up locally to do just that.

More information about the Okta IDX API will be documented soon.

### CODE REVIEW

File: docs/okta/idx/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+You'll also need an Hoppscotch "Environment" containing the variables/secrets.
+
+A collection pointing to the CODE environment is available on S3. Use the following to download it. Be sure to have the `identity` Janus credentials.
+
+```sh
+# this downloads it to your home directory, feel free to change the path if needed e.g. to your downloads folder
+$ aws s3 cp --profile identity s3://identity-private-config/DEV/identity-gateway/okta_code_environment_hoppscotch.json ~/okta_code_environment_hoppscotch.json
+```
+
+Once downloaded, in Postman you can:
```
Comment (AshCorr) [inline docs/okta/idx/README.md:30]:
```suggestion
Once downloaded, in Hoppscotch you can:
```

Comment (coldlink) [reply docs/okta/idx/README.md:30]: 🤦🏾

File: docs/okta/idx/README.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+Under the "secrets" tab, these are all things which are either secret, or change frequently. These should be filled in as needed.
+
+### Using the collection
+
+To interface with the Okta IDX API, you always have to make 2 calls in order to begin with.
+
+1. First call the "IDX /interact" endpoint.
+   - All the variables/parameters should be set up for this, so just click send!
+   - This returns an `interaction_handle` in the body.
+   - Copy this value, and add it to the `interactionHandle` secret under the environment
```
Comment (AshCorr) [inline docs/okta/idx/README.md:50]: Hoppscotch can probably automate this using post-request scripts: https://docs.hoppscotch.io/documentation/getting-started/rest/tests

I seem to remember being able to do something like this in Postman!

File: docs/okta/idx/README.md (hunk 1 above)
Comment (coldlink) [reply docs/okta/idx/README.md]: Updated!

**Review by AshCorr — APPROVED** [review]

File: docs/okta/idx/README.md (hunk 2 above)
Comment (coldlink) [reply docs/okta/idx/README.md:50]: Oh good shout. I'll see if this is possible, and can always update the collection with added scripts in a separate PR.

Comment (AshCorr) [reply docs/okta/idx/README.md:50]: Ah they have an example on doing this here https://docs.hoppscotch.io/documentation/features/scripts#setting-environment-variables-from-api-responses

Comment (coldlink) [reply docs/okta/idx/README.md:50]: https://github.com/guardian/gateway/pull/2856

**Pull request merged**